xpcs_online_boost_client.py

```diff
@@ -84,7 +84,7 @@ def arg_parse():
     #source_raw_path = os.path.join(args.experiment, 'data', args.raw)
     #source_qmap_path = os.path.join(args.experiment, 'data', args.qmap)
     #Output path to transfer results back
-    result_path = os.path.join(args.output_dir, hdf_name)
+    # result_path = os.path.join(args.output_dir, hdf_name)
```

Comment:
The new changes require adding this "output_dir" flag. If not set, we can't do the result_transfer part. I'm not quite sure what this value should be for testing. I think it will also overwrite source files (although I don't think the input files are changed by corr).

Reply:
This helps a lot, and I think it answers a question I had from earlier. So then the output dir for all datasets is given by the above custom name `analysis`, and the dirname and HDF input filename stay the same? Additionally, confirming that this is a single file transfer of only the output hdf. 

Thakns, that's something that can be tested and checked.